cookbooks/ey-core/recipes/swap.rb

```diff
@@ -2,6 +2,9 @@
   code <<-EOH
     parted -s -a optimal /dev/xvdc mklabel msdos
     parted -s -a optimal -- /dev/xvdc unit compact mkpart primary linux-swap "1" "-1"
+    while [[ ! -b /dev/xvdc1 ]]; do
+      sleep 1
```

Comment:
I have no problem with this, it's a sensible change - but has this ever actually broken?? I don't recall ever seeing it fail!

Reply:
Ahh, this makes sense!